composer.json

```
@@ -22,7 +22,6 @@
         "friends-of-behat/variadic-extension": "^1.3",
         "nelmio/alice": "^3.10",
         "nyholm/psr7": "^1.8",
-        "phpspec/phpspec": "^7.0",
         "phpunit/phpunit": "^9.5",
```

```suggestion
        "phpunit/phpunit": "^10.5",
```
to be consistent with Sylius